assets/src/edit-story/components/reorderable/reorderableSeparator.js

```diff
@@ -44,22 +44,31 @@ const Line = styled.div`
 `;
 
 function ReorderableSeparator({ position, ...props }) {
+  const separatorRef = useRef(null);
+
   const {
     actions: { setCurrentSeparator },
     state: { isReordering },
   } = useReorderable();
   const handlePointerEnter = useCallback(() => {
     setCurrentSeparator(position);
+    separatorRef.current.scrollIntoView({
```

Comment:
Safari only supports `scrollIntoView` with boolean parameter, but not smooth behavior option.

Reply:
That should be alright, just a slightly worse experience. Otherwise we can revert back to @barklund 's approach which should work for all browsers